ClavaLaraExtraApi/.project

```diff
@@ -1,28 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Keep file

Reply:
Will be moved to a new branch and not reapplied to the main project. This is to prevent dead-code existing or being merged into staging or master.

Reply:
Resolved in 44508bf26 with the creation of the `ClavaLaraExtraAPIs` branch.